service/windows/src/main.rs

```diff
@@ -0,0 +1,312 @@
+#[macro_use]
+extern crate windows_service;
+
+mod config;
+mod manager;
+mod node;
+mod subprocess;
+
+use crate::config::{get_config_file_path, load_config};
+use crate::subprocess::create_console;
+use anyhow::{bail, Context};
+use log::*;
+use retain_mut::RetainMut;
+use std::ffi::OsString;
+use std::string::String;
+use std::sync::{
+    atomic::{AtomicBool, Ordering},
+    Arc,
+};
+use std::time::{Duration, Instant};
+use std::{env, thread};
+use winapi::shared::winerror::ERROR_FAILED_SERVICE_CONTROLLER_CONNECT;
+use windows_service::{service::*, service_control_handler::*, service_dispatcher};
+
+const SERVICE_NAME: &str = "ConcordiumNode";
```

Comment:
Why not just "Concordium Node"? Or is the UpperCamelCase the standard way of naming services?

Reply:
This seems to be the convention. There are exceptions, but very few services include spaces in their names. Note that the service also has a display name, which is how it's listed in the services panel. (Though the service name is used in Task Manager.)